roles/pulp_content_guard/tasks/rbac/rbac.yml

```diff
@@ -0,0 +1,99 @@
+---
+
+- name: Get RBAC content guard list
+  uri:
+    url: "{{ pulp_rbac_cg_url }}"
+    user: "{{ pulp_username }}"
+    password: "{{ pulp_password }}"
+    method: GET
+    status_code: 200
+    force_basic_auth: true
+  no_log: true
+  register: rbac_cg_list_result
+
+- name: Set fact remove_rbac_cg
+  set_fact:
+    remove_rbac_cg: "{{ (remove_rbac_cg | default([])) + [item.name] }}"
```

Comment:
Safest to initialise this variable to an empty list, in case the role is run multiple times in the same ansible run.